userguide/config.toml

```diff
@@ -97,14 +97,19 @@ time_format_default = "January 2, 2006"
 rss_sections = ["blog"]
 
 # Google Custom Search Engine ID. Remove or comment out to disable search.
-gcs_engine_id = "011217106833237091527:la2vtv2emlw"
+#gcs_engine_id = "011217106833237091527:la2vtv2emlw"
+
+# Enable Algolia DocSearch
+algolia_docsearch = true
```

Comment:
Yes, add this to the example project as well (with both set to "false", though I actually like the Algolia results so much I'm toying with the idea of making them the default...)